middleware.ts

```diff
@@ -9,7 +9,7 @@ import { pathToRegexp } from "path-to-regexp";
 // prettier-ignore
 const pathsToMatch = [
   "/",
-  "/hub/insights/:path*",
+  // "/hub/insights/:path*",
```

Comment:
refactor (blocking): If the code is commented out, remove it. We can always revert if we need it back.